.evergreen/atlas_data_lake/config.yml

```diff
@@ -3,7 +3,7 @@ environment: local
 showInternalErrors: true
 
 frontend:
-  bindAddrs: ["localhost:27017"]
+  bindAddr: "localhost:27017"
```

Comment:
Is `bindAddr` is a new config key introduced by [MHOUSE-1860](https://jira.mongodb.org/browse/MHOUSE-1860)? Is `bindAddrs` being kept in place for BC or removed in favor of this new key?

IIRC, [`build-mongohouse-local.sh`](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/atlas_data_lake/build-mongohouse-local.sh) builds the master branch of mongohouse. Do I assume correctly that you'd plan to merge this PR immediately after MHOUSE-1860 is merged to avoid a situation where our config file fails to parse?

The change itself LGTM.

Reply:
> Is `bindAddr` is a new config key introduced by [MHOUSE-1860](https://jira.mongodb.org/browse/MHOUSE-1860)? Is `bindAddrs` being kept in place for BC or removed in favor of this new key?

It's been removed.

> IIRC, [`build-mongohouse-local.sh`](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/atlas_data_lake/build-mongohouse-local.sh) builds the master branch of mongohouse. Do I assume correctly that you'd plan to merge this PR immediately after MHOUSE-1860 is merged to avoid a situation where our config file fails to parse?

The mongohouse repo already has this change (was merged yesterday).  And, yes, the order of these commits was intentional so as to avoid parsing issues on your end. :)

> The change itself LGTM.

Thanks!